.github/workflows/format.yml

```diff
@@ -44,7 +44,7 @@ jobs:
     - name: Configure
       run: >
         cmake -S . -B build
-        -DCMAKE_CXX_CLANG_TIDY="$(which clang-tidy)"
+        -DCMAKE_CXX_CLANG_TIDY="$(which clang-tidy);--use-color;--warnings-as-errors=*"
```

Comment:
@Skylion007 Warnings as errors is set here. It's more inconvenient running this locally with warnings as errors, since it stops the compilation (requiring `--keep-going`, for example). Notice the "local run" text changes too. It's better to have this as a user-control via `CMAKE_CXX_CLANG_TIDY` vs. always forcing it. (IMO)

Reply:
Do you like it then, or would you rather it the old way?